ui/src/translations/IntlGlobalProvider.js

```diff
@@ -36,10 +36,13 @@ class IntlTranslator {
   // Formatting Functions
   // ------------------------------------
   translate(key, values) {
+    if (!key) {
+      return '';
+    }
```

Comment:
this is the only relevant change, the rest of the diff is linting/cosmetics and belong to a dedicated commit